schema/meta.schema.yaml

```diff
@@ -40,10 +40,12 @@ properties:
       - 133
       - 134
       - 135
+      - 136
   fonts:
     type: array
     items: { type: string }
   compatibilityVersion: { const: 99 }
+  coeditCompatibilityVersion: { type: number }
```

Comment:
Probably this should be `optional`, otherwise the schemas won't validate files without it, despite Sketch being happy to open them.

Reply:
Good point, updated